src/charm.py

```diff
@@ -79,6 +82,8 @@ def __init__(self, framework: ops.framework.Framework) -> None:
         self.framework.observe(self.on.install, self._on_install)
         self.framework.observe(self.on.stop, self._on_stop)
         self.framework.observe(self.on.set_pool_size_action, self._set_pool_size_action)
+        self.framework.observe(self.on.exit_maintenance_action, self._exit_maintenance_action)
```

Comment:
why add the new event handlers in the middle of existing ones ? in my view, this should be done at the end of existing code possibly with a line break in-between to differentiate the action handlers visually.

Reply:
Move down to the end of existing actions

Reply:
ack, thanks